src/config.js

```diff
@@ -48,6 +48,7 @@ const SCHEDULE_MESSAGES = {
     ? csvParser(process.env.SCHEDULED_MESSAGES)
     : [],
   timeZone: process.env.TIME_ZONE,
+  path_markdown_folder: '../../../markdown-files',
```

Comment:
I think messageTimes would not be used anymore, since the markdown files are the new way to create automatic messages https://github.com/TeamNovaSoft/discord-bot/pull/83/files/289e624e98faf5b9670e9f477594773236fab5b4..3cd424b8604652781cbce4a5c837ba61c8381a54#diff-23821ae067a9c7dfb6227774aefb39fa3951a5874ea953b9401dfc80175c026cR47